src/ledger/ledger-wrapper.ts

```diff
@@ -76,17 +78,25 @@ export const encodeDerivationPath = (derivationPath: string) => {
   return `${dataLength}${data}`
 }
 
+const LedgerSubjects = () => ({
+  onProgressSubject: new Subject<string | undefined>(),
```

Comment:
what's the point of sending a undefined progress event?

Reply:
When exchange finishes with error I'm sending `undefined` in order to visually get back to initial state

Reply:
I guess I can send empty string if it'd be better for you